src/sass/style.scss

```diff
@@ -1,8 +1,9 @@
-@import "reset";
-@import "variables";
-@import "typography";
-@import "base";
-@import "header";
-@import "carousel";
-@import "circle-item";
-@import "slider-item";
+@import 'reset';
+@import 'variables';
+@import 'typography';
+@import 'base';
+@import 'header';
+@import 'carousel';
+@import 'circle-item';
+@import 'slider-item';
+@import 'footer';
```

Comment:
In most IDEs you have an option to sort imports on save. I would highly recommend that :)